# Application submission info

---
> Submitted on 20-Mar-2024, approved 21-Mar-2024.
---

Submitted through ACCESS: https://access-ci.org/

Submitted an "Explore ACCESS", different types listed here: https://allocations.access-ci.org/project-types

After logged in, submission process started from this page: https://allocations.access-ci.org/opportunities

Then selecting to "Request New Project", and choosing "Request an Explore ACCESS project". 


## Required information I added to the form

**Title:** NASA JPL 2024 Preparing for Research in Space Microbiology (PRISM) Program

**Public overview:**  

> General overview
>
> In a Historically Black Colleges and Universities (HBCU) and Minority Serving Institutions program, we are working with students and educators from several institutions including: California State University, Northridge; California State Polytechnic University, Pomona; and Alabama State University. Over different days, each group will be going through the process of extracting DNA from microbial isolates recovered from spacecraft-assembly clean rooms at NASA JPL, and sequencing them, and then we will be teaching/doing the associated bioinformatics work with the students where they will be assembling the genome they had extracted. 
> 
> How we plan to use ACCESS resources
> 
> We will be running a 1-day, remote computational workshop where we will be going through the concepts and process of assembling our isolate genomes, and then allowing them access to the cloud environments for about a week following the workshop. I have been fortunate enough to receive XSEDE resources for many workshop/education scenarios over the past, and it has been invaluable. 
> 
> I have experience in the past doing this through Indiana Jetstream2, managing through Exosphere, so that would be great if possible. We are requesting the ability to run about 40 m3.large instances concurrently for about 10 days straight, which based on the Jetstream estimator is about 154,000 SUs. So the starting 200,000 credits that come along with an Explore ACCESS allocation will be more than sufficient to enable us to run this course. Please let me know if should be estimating a different way or if any other information is needed. 
> 
> Thanks for your consideration and any help!


**Keywords:** space biology, bioinformatics

**Grant Type:** Other

**Opportunity questions:** Didn't check any as none applied currently.

**Fields of Science:** â€œOther Biological Sciencesâ€

**Documents:** Attached my CV here.

**Available resources:** Checked box for ACCESS Credits. 

---

## After getting accepted

### Transferring credits from ACCESS to Jetstream2
Once approved, and logged in, needed this page (https://allocations.access-ci.org/requests) in order to transfer ACCESS credits to a specific resource. For the appropriate allocation/Project, selected "Credits + Resources", then the text box that initially says "Add a resource to your exchange...", then selected "Indiana Jetstream2 CPU", then entered 196,000. Then for "Indiana Jetstream2 Storage" added the remaining 4,000, giving 4 TB of shared storage to use.

### Requesting increase in quotes
The allocation also comes with a limit on the number of concurrent instances that can be run. I submitted a request when logged into Jetstream2 here: https://jetstream2.exosphere.app/exosphere/getsupport

This is the text I submitted:

> Hi there :)
>
> We plan to use this allocation (BIO240084) with around 45 concurrent m3.large instances for a bioinformatics course we are running. The starting limits cap us at 10.
> 
> Could you please help with increasing the alloted quotas so that we will be able to run up to 45 m3.large instances concurrently on this allocation, including cores, ram, volume, ports, available IP addresses, and whatever other magic you folks take care of that i'm naive to?
> 
> Thank you for any help!
> -Mike

---
